frontend/packages/schema-editor/src/components/SchemaInspector/NameField.tsx

```diff
@@ -57,16 +57,15 @@ export function NameField({ id, pointer, handleSave, label, ...props }: NameFiel
             return '';
         }
       }}
-    >
-      {({ errorCode, onChange, customRequired }) => (
+      renderField={() => (
         <AltinnTextField
           id={id}
-          onChange={(e) => onChange(e.target.value, e)}
-          onBlur={(e) => onNameBlur(e.target.value, errorCode)}
-          withAsterisk={customRequired}
+          onChange={(e) => setNodeName(e.target.value)}
+          onBlur={(e) => onNameBlur(e.target.value, '')}
+          withAsterisk={true}
           {...props}
         />
       )}
```

Comment:
Er det noen grunn til at du ikke bruker props fra `renderField` her? Du har fortsatt tilgang de samme verdiene der. Alt du trenger å endret er å flytte feltet inn i `renderField` og legge til `...rest` for å få med alle egenskapene.
```
      renderField={({errorCode, onChange, customRequired, ...rest}) => (
        <AltinnTextField
          {...rest}
          id={id}
          onChange={(e) => onChange(e.target.value, e)}
          onBlur={(e) => onNameBlur(e.target.value, errorCode)}
          withAsterisk={customRequired}
        />
      )}
```
Jeg husker ikke hvorfor vi fjernet `errorCode`, men vi trenger den tydeligvis her, så det er bare å sette den tilbake igjen.